client/index.js

```diff
@@ -6,10 +6,14 @@ import { store } from './store';
 import App from './App.jsx';
 // reminder to get scss or css in here
 import './styles/login.css';
+import { BrowserRouter } from 'react-router-dom';
 
 render(
   <Provider store={store}>
-    <App />
+    <BrowserRouter>
```

Comment:
Everything looks good! The only thing is in the index.js, since Router is already inside the app.jsx, we don't need it elsewhere.